libpages/site.go

```diff
@@ -0,0 +1,77 @@
+// Copyright 2017 Keybase Inc. All rights reserved.
+// Use of this source code is governed by a BSD
+// license that can be found in the LICENSE file.
+
+package libpages
+
+import (
+	"context"
+	"net/http"
+	"os"
+	"sync"
+	"time"
+
+	"github.com/keybase/kbfs/libfs"
+	"github.com/keybase/kbfs/libpages/config"
+)
+
+const configCacheTime = 16 * time.Second
+
+type site struct {
+	// fs should never once constructed.
```

Comment:
"never once"?

Reply:
Whoa how did that happen. Should've been "fs should never be changed once it's constructed".